SofaKernel/modules/SofaSimulationCore/src/sofa/simulation/fwd.h

```diff
@@ -55,3 +55,8 @@ SOFA_SIMULATION_CORE_API sofa::core::objectmodel::BaseContext* toBaseContext(Nod
 SOFA_SIMULATION_CORE_API Node* getNodeFrom(sofa::core::objectmodel::Base*);
 SOFA_SIMULATION_CORE_API Node* getNodeFrom(sofa::core::objectmodel::BaseContext*);
 }
+
+namespace sofa::simulation::common
+{
+    class MechanicalOperations;
+}
```

Comment:
Should be in SofaSimulationCommon module?

Reply:
Despite its namespace MechanicalOperations.h is in SofaSimulationCore. So until this is fixed it seems ok have its in this SoaSimulationCore/fwd.h